minichain/transaction.py

```
@@ -35,18 +36,18 @@ def __init__(self, sender, receiver, amount, nonce, data=None, signature=None, t
-        return {"sender": self.sender, "receiver": self.receiver, "amount": self.amount,
+        return {"sender": self.sender, "receiver": self.receiver, "amount": self.amount, "fee": self.fee,
                 "nonce": self.nonce, "data": self.data, "timestamp": self.timestamp,
                 "signature": self.signature}
 
     def to_signing_dict(self):
-        return {"sender": self.sender, "receiver": self.receiver, "amount": self.amount,
+        return {"sender": self.sender, "receiver": self.receiver, "amount": self.amount, "fee": self.fee,
                 "nonce": self.nonce, "data": self.data, "timestamp": self.timestamp}
 
     @classmethod
     def from_dict(cls, payload: dict):
```

Ruff (0.15.15)

[warning] Missing return type annotation for classmethod `from_dict`

(ANN206)